README.md

```diff
@@ -5,6 +5,12 @@ The main purpose of this module is to provide a PuppetDB import source
 for Icinga Director. As of this writing it has not even a GUI or CLI
 component, but this may change in future.
 
+To install, please refer to our documentation. [Installation and configuration](doc/01-Installation.md)
+Note: Some steps require you to be familiar with Puppet/PuppetDB. 
+
+
+==========
+
+Screnshot:
```

Comment:
NB: And google for git squash ;-) It's a very interesting feature, you really should try to figure out how it works. But in case you do not want to waste that much time for such a small change don't worry, just let me know. I can fetch the patch and push it in your name if you want me to do so.